packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Microsoft.Generator.CSharp.ClientModel.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Can we open a tracking issue to move framework / nullable etc into a common Build.Directory.props for all projects to share.

Reply:
Opened https://github.com/microsoft/typespec/issues/3393